hardware/1Wire/1WireByKernel.cpp

```diff
@@ -233,8 +233,11 @@ void C1WireByKernel::GetDevices(/*out*/std::vector<_t1WireDevice>& devices) cons
 {
    // If devices not initialized (ie : at least one time read), return an empty list
    // to not update Domoticz with inconsistent values
-   if (!m_AllDevicesInitialized)
-      return;
+
+	while (!m_AllDevicesInitialized)
```

Comment:
what is 'm_AllDevicesInitialized' never becomes true ? this is a deadlock situation then ?
It seems GetDevices is only called once... 
Maybe this should be done in the poll interval (1Wire.cpp, line 139) until m_AllDevicesInitialized is true ?